pkg/controller/revision/revision_controller.go

```diff
@@ -168,14 +168,27 @@ func (r *ReconcileRevision) Reconcile(request reconcile.Request) (reconcile.Resu
 		}
 	}
 
-	triggered, err := r.promAPI.IsRevisionTriggered(context.TODO(), instance.Spec.App.Name, instance.Spec.App.Tag)
+	triggered, err := r.promAPI.IsRevisionTriggered(context.TODO(), instance.Spec.App.Name, instance.Spec.App.Tag, instance.Spec.CanaryWithSLIRules)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
 	if triggered && !instance.Spec.IgnoreSLOs {
 		accepted := true
+		var acceptanceTargets []string
```

Comment:
what are acceptance targets?

Reply:
Until this PR picchu will not canary anything but hardcoded `production` targets. This allows any target to define itself as an acceptanceTarget and enable canarying.

Reply:
I'm going to rename this to `canaryTargets` I think.

Reply:
🤷‍♀ I kind of like acceptance targets, now that I get it. the notion could easily expand to techniques beyond canarying.